index.js

```diff
@@ -331,3 +605,59 @@ function destroy() {;
   return Promise.all(promises);
 }
 
+// Lookup the conversations for the calls to get their title
+async function lookupConversations(client, calls) {
+  let promises = [];
+  calls.forEach(call => {
```

Comment:
The SDK has an api for this getConversationsByIds which can look up the conversations of an array of convIds. You should probably use that rather than make multiple api calls like this.
https://circuitsandbox.net/sdk/classes/Client.html